### PR TITLE
datasources/items.js - Add getByBsgCategoryId to get a list of items based on their bsgCategoryId

### PR DESCRIPTION
**Pull request by NicolaGenesin**

### CODE REVIEW

File: datasources/items.js [hunk 1]
```diff
@@ -184,6 +184,16 @@ class ItemsAPI {
             return this.formatItem(rawItem);
         });
   }
+
+  getByBsgCategoryId = (bsgCategoryId) => {
```
Comment (kokarn) [inline datasources/items.js]: Should probably follow the rest of the file
```
getByBsgCategoryId(bsgCategoryId) {
```

**Comment by kokarn** [comment]
You haven't updated resolvers & the schema file, so this will not actually be exposed anywhere yet :)

File: datasources/items.js (hunk 1 above)
Comment (NicolaGenesin) [reply datasources/items.js]: ✅

**Review by kokarn — APPROVED** [review]

**Comment by kokarn** [comment]
Looks good!

**Pull request merged**